run.sh

```diff
@@ -9,6 +9,7 @@
 # To test that try running `docker run --rm --env-file .env busybox /bin/sh -c 'echo $NODE_ECDSA_KEY_PASSWORD'`
 # This will output password with single quote. Not sure why this happens.
 optIn() {
+  ./srs_setup.sh
```

Comment:
should it shortcircuit if this script fails?

Reply:
yea I guess we should not opt-in if anything - downloading or validating SRS points fails